discord_bots/commands.py

```diff
@@ -1260,9 +1261,11 @@ async def cancelgame(ctx: Context, game_id: str):
             if guild_channel:
                 await guild_channel.delete()
         session.delete(channel)
+    session.commit()
 
     session.delete(game)
     session.commit()
+    session.close()
```

Comment:
we shouldn't close the session here because this function doesn't own it, it was passed in
```suggestion
```

Reply:
session.close() removed on next commit 983e932